nasa/openmct
# [Clock] Move Angular Clock and Clock Indicator to Vue

Fixes #4042  

### Author Checklist

Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | Y
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

### Testing instructions
1. If possible, create a clock before the change and then change to the branch with the change. Ensure that the clock continues to display properly and that editing the properties works.
2. Create a clock object and choose some values in the creation dialog. Ensure that the time is displayed and is in the appropriate format based on the options chosen (proper timezone, format, 12/24 hours).
3. Edit the properties to modify the display format and ensure the clock updates with the correct time.
4. Edit the properties to modify the 12/24 hours selection and ensure the clock updates with the correct time.
5. Edit the properties to modify the timezone and ensure the clock updates with the correct time.
6. Check that the clock indicator is displayed when the option `enableClockIndicator` is set to `true` in index.html and displays time in UTC.
7. It would be useful to make sure the timer still works properly as well since code around it was removed for this change.
